CalibMuon/DTDigiSync/plugins/SealModule.cc

```diff
@@ -0,0 +1,9 @@
+#include "FWCore/PluginManager/interface/PluginFactory.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncFactory.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncTOFCorr.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncT0Only.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncFromDB.h"
```

Comment:
if these files are used only in `CalibMuon/DTDigiSync/plugins`, isn't our more common pattern to have the declaration and implementation in /plugins ?

Reply:
.. I meant to refer to `DTTTrigSyncTOFCorr`, `DTTTrigSyncT0Only`, and `DTTTrigSyncFromDB` (not the Factory, which should be in the public interface)

Reply:
you are right, nothing else is using these headers so moving them to plugins is the right thing to do

Reply:
https://github.com/cms-sw/cmssw/pull/34853 moves these headers and src files in to plugins directory